### PR TITLE
Rewrite runner

### LINKED ISSUE #4: Write tests

### LINKED ISSUE #8: Time tests
Print total time and time for each task

### LINKED ISSUE #9: Display coverage

### PR DESCRIPTION
**Pull request by eirsyl**
- Change design, use clint for commandline functions
- Only print tasks with error by default
- Print project coverage
- Display task execution time
- Write tests
- Fixes #9
- Fixes #8
- Fixes #4

### CODE REVIEW

File: tests/test_runner.py [hunk 1]
```diff
@@ -1 +1,204 @@
...
+        projects.build_settings.assert_called_once_with(runner.directory)
+
+    def test_no_tasks(self):
+        """
+        No tasks, system exit.
+        """
+        def raise_runtime_error(*args, **kwargs):
+            raise RuntimeError
+
+        projects.build_settings = mock.Mock(side_effect=raise_runtime_error)
```
Comment (relekang) [inline tests/test_runner.py]: Please use mock.patch-decorator instead. The side_effect argument can take an exception class or instance and raise it so `side_effect=RuntimeError` should give the same result.

File: tests/test_runner.py [hunk 2]
```diff
@@ -1 +1,204 @@
...
+        """
+        Test init of the runner class
+        """
+        runner = Runner(True, True)
+
+        self.assertTrue(runner.fail_fast)
+        self.assertTrue(runner.verbose)
+        self.assertEqual(runner.directory, os.getcwd())
+
+        projects.build_settings.assert_called_once_with(runner.directory)
```
Comment (relekang) [inline tests/test_runner.py]: `mock_run.assert_called_once_with(runner.directory)`

File: tests/test_runner.py [hunk 3]
```diff
@@ -1 +1,204 @@
...
+from invoke.exceptions import Failure
+from invoke.runner import Result
+
+from frigg_runner.runner import Runner
+
+
+class RunnerTestCase(unittest.TestCase):
+
+    @mock.patch('frigg.projects.build_settings')
+    def test_runner_init(self, mock_run):
```
Comment (relekang) [inline tests/test_runner.py]: Please use descriptive names for mocks, like `mock_build_settings`.

**Comment by relekang** [comment]
Looks great :sparkles:

**Comment by eirsyl** [comment]
image

New design

**Comment by eirsyl** [comment]
:rocket:

**Pull request merged**